services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/operations/ScanOperation.java

```diff
@@ -82,6 +82,7 @@ public ScanRequest generateRequest(TableSchema<T> tableSchema,
             .exclusiveStartKey(this.request.exclusiveStartKey())
             .consistentRead(this.request.consistentRead())
             .returnConsumedCapacity(this.request.returnConsumedCapacity())
+            .select(this.request.select())
```

Comment:
In case the "select" enum value isn't known to the SDK version, it would be nice to use 'String' internally, like the code-generated models do. That way, customers that can't upgrade their SDK version can still specify new "select" options we might add in the future.

I see we're not doing that for other enums, and I don't think "select" changes much, so "in a separate PR" is totally fine.

Reply:
I think we should do as part of separate PR for all the Operations where select is used. AFAIK its for Query and Scan.